modules/announcements.lua

```diff
@@ -109,8 +124,10 @@ end
 -- Sends an announcement
 -- Param unit is only used for class coloring of messages
 function Announcements:Send(msg, throttle, unit)
-    -- only send announcements inside arenas
-    if select(2, IsInInstance()) ~= "arena" then return end
+    -- only send announcements inside arenas and battlegrounds
+    if select(2, IsInInstance()) ~= "arena" or select(2, IsInInstance()) ~= "pvp" then return end
```

Comment:
This will be rejected. GladiusEx is an arena addon.

Reply:
Also, this is incorrectly formatted and will fire neither in arena nor bgs.

Reply:
Is it possible to add some kind of sapped announcement? My fork doesnt work idk how to edit your code